src/sklearn.jl

```diff
@@ -238,4 +238,35 @@ function generate_classification(;  n_samples::Int = 100,
                                                         random_state = random_state)
 
     return convert(features, labels)
-end
\ No newline at end of file
+end
+
+
+"""
+function generate_low_rank_matrix(; n_samples::Int =100,
+                                    n_features::Int =100,
+                                    effective_rank::Int =10,
+                                    tail_strength::Float64 =0.5,
+                                    random_state::Union{Int, Nothing} = nothing)
+Generate a mostly low rank matrix with bell-shaped singular values
+#Arguments
+- `n_samples::Int = 100`: The number of samples.
+- `n_features::Int = 20`: The total number of features. These comprise `n_informative` informative features, `n_redundant` redundant features, `n_repeated` duplicated features and `n_features-n_informative-n_redundant-n_repeated` useless features drawn at random.
+- `effective_rank::Int = 10`: The approximate number of singular vectors required to explain most of the data by linear combinations.
+- `tail_strength::Float64 = 0.5`: The relative importance of the fat noisy tail of the singular values profile.
+- `random_state::Union{Int, Nothing} = nothing`: Determines random number generation for dataset creation. Pass an int for reproducible output across multiple function calls. See Glossary.
+Reference: [link](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_low_rank_matrix.html)
+"""
+function generate_low_rank_matrix(; n_samples::Int = 100,
+                                    n_features::Int = 100,
+                                    effective_rank::Int = 10,
+                                    tail_strength::Float64 = 0.5,
+                                    random_state::Union{Int, Nothing} = nothing)
+
+    (features, labels) = datasets.make_low_rank_matrix(n_samples = n_samples,
+                                                       n_features = n_features,
+                                                       effective_rank = effective_rank,
+                                                       tail_strength = tail_strength,
+                                                       random_state = random_state)
+   #return convert(features, labels)
```

Comment:
Por que tá comentado?

Reply:
na verdade, esqueci de alterar ali na linha 265, mas o retorno da função low_level_matrix é único. 
```
Returns:
X: array of shape [n_samples, n_features] (The matrix).
``` 
Então pelo que entendi não precisa aplicar o convert né? (ele também da um erro, se eu aplico)

```
MethodError: no method matching convert(::Float64, ::Float64)
```

Reply:
se for preciso eu faço uma alteração maior lá

Reply:
Nesse caso não. Ele não é um problema de classificação. Acho melhor retornar a matriz mesmo.